.changeset/early-stingrays-brake.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-devtools-backend': patch
+'@backstage/plugin-devtools': patch
+---
+
+Update devtools info in real time using signals
```

Comment:
Looks good to me. I will be on holiday for a week now but will get back to this then (if you don't have time to push this forward). This requires the permission support to signals though; otherwise anyone can subscribe to these updates.